src/index.css

```
@@ -116,6 +181,36 @@ h1 {
     margin-left: 24px;
     margin-right: 24px;
   }
+
+  nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
+    align-items: center;
+    height: 74px;
+    margin-left: 24px;
+    padding-bottom: 0;
+    padding-top: 0;
```

How about

```suggestion
    padding: 0;
```

Or is there padding that would be overridden by that which you would like to avoid?